docs/configuration/registration/objects.rst

```diff
@@ -0,0 +1,52 @@
+.. _configuration_registration_objects:
+
+============
+Objecten API
+============
+
+The Objecten API is a suite of REST based object registration services. Open Forms can be
+configured to access these API's to register form submissions.
+
+1. In Open Forms, navigate to: **Configuration** > **Services**
+2. Here we need to configure credentials for two related API's:
+
+    - Objecten API (ORC)
+    - Documentregistratiecomponent (DRC)
+
+   a. Click **Service toevoegen**.
+   b. Fill out the form for each of the two components:
+
+      * **Label**: *For example:* ``Objecten`` or ``Documenten``
```

Comment:
Can you split these in 2 separate steps? And explain that the submission of and attachments are stored in the documenten API